ReqIFSharp/Datatype/DatatypeDefinitionInteger.cs

```diff
@@ -20,80 +20,81 @@
 
 namespace ReqIFSharp
 {
-    using System.Globalization;
-    using System.Xml;
+	using System.Xml;
```

Comment:
code looks good, thanks for the fix. Just an issue with the styles. Seems like too many tabs and spacing around parameters etc. Can you update please so the style is similar to the entire code-base?

Reply:
No so easy as this forces me to change my Visual Studio and Resharper coding style settings.
You do not happen to have a suitable .editorconfig file for this project?